### PR TITLE
SFR-914: Add author and work title to edition page

### PR DESCRIPTION
**Pull request by jackiequach**
[Jira Ticket](http://jira.nypl.org/browse/SFR-914)

### This PR does the following:
-

### Testing requirements & instructions: 
-

### CODE REVIEW

File: src/util/EditionCardUtils.tsx [hunk 1]
```diff
@@ -107,6 +107,30 @@ export default class EditionCardUtils {
     });
   }
 
+  static getAuthorsListByName(authors: string[]): JSX.Element[] {
```
Comment (jackiequach) [inline src/util/EditionCardUtils.tsx:110]: Is viaf still being used? If not, we should remove the above method.

Comment (hokei) [reply src/util/EditionCardUtils.tsx:110]: Viaf is still working.  If you click on an author who has a viaf, it does a search for "viaf" but displays the author name - you can see it when you click "Shakespeare" under https://digital-research-books-beta.nypl.org/work/023b92c2-c371-448c-8a10-4a9272ae0dee?featured=8576298

File: src/components/EditionDetail/Edition.tsx [hunk 2]
```diff
@@ -107,6 +114,9 @@ const Edition: React.FC<{ editionResult: EditionResult; backUrl?: string }> = (
             )}
           </Flex>
           {edition.sub_title && <Box>{edition.sub_title}</Box>}
+          {authorsList && authorsList.length && (
+            <Box>By {joinArrayOfElements(authorsList, ", ")}</Box>
```
[the remark this answers is not in the document]
Comment (jackiequach) [reply src/components/EditionDetail/Edition.tsx:118]: A `<Box>` renders as a `<div>` by default, but you can also render it another element using the `as` prop. It also allows us to add styles as necessary.

File: src/util/EditionCardUtils.tsx (hunk 1 above)
Comment (hokei) [reply src/util/EditionCardUtils.tsx:110]: you're probably going to want to add that to the edition detail pages as well - as you can seek if you click "Shakespeare" under https://sfr-bookfinder-front-end-git-sfr-914editionpageauthortitle-nypl.vercel.app/work/b9a174db-bf1f-4326-8270-8716117923e2?featured=6173984 it dies a slightly different search than if you click under the edition: https://sfr-bookfinder-front-end-git-sfr-914editionpageauthortitle-nypl.vercel.app/edition/6173984?featured=8861813

**Comment by hokei** [comment]
Looks like there's a bug in the item details page:  https://sfr-bookfinder-front-rnz7ljhhs-nypl.vercel.app/edition/6173541?featured=8861665

**Review by hokei — CHANGES_REQUESTED** [review]
One last bug, I think everything else looks good.

**Comment by mwbenowitz** [comment]
> Looks like there's a bug in the edition details page: https://sfr-bookfinder-front-rnz7ljhhs-nypl.vercel.app/edition/6173541?featured=8861665

This was due to an odd error where some changes failed to deploy to the QA API. Just forced an update there and it looks like it's fixed!

**Pull request merged**